pkg/build/builder/dockerutil.go

```diff
@@ -110,8 +110,15 @@ func pullImage(client DockerClient, name string, authConfig docker.AuthConfigura
 	logProgress := func(s string) {
 		glog.V(0).Infof("%s", s)
 	}
+	tag := "latest"
```

Comment:
i think ParseDockerImageReference giving you DockerImageReference  that has Tag is your best bet :-)